ci/pipeline.yml

```diff
@@ -0,0 +1,152 @@
+---
+groups:
+  - name: bosh-oracle-cpi-release
+    jobs:
+      - test-unit
+      - build-candidate
+      - setup-infrastructure
+      - deploy-ubuntu
+      - run-bats
+
+jobs:
+  - name: test-unit
+    plan:
+      - aggregate:
+        - {trigger: true, get: bosh-cpi-src, resource: bosh-cpi-src-in}
```

Comment:
Don't we need another resource and a trigger to pull cpi-sources?

Reply:
By cpi-sources, do you mean a separate pull of the submodule bosh-oracle-cpi?

Reply:
Yes, that's what I meant.

Reply:
Currently, pulling bosh-oracle-cpi-release, also checks out bosh-oracle-cpi. If the pipeline isn't triggered by checkins to bosh-oracle-cpi, then we can add the trigger